service/controller/controllercontext/status_types.go

```diff
@@ -53,6 +53,7 @@ type ContextStatusTenantClusterEncryption struct {
 
 type ContextStatusTenantClusterMasterInstance struct {
 	DockerVolumeResourceName string
+	EtcdVolumeSnapshotID     string
```

Comment:
@lachlanh I added this as I need this when creating the snapshot during upgrades, because we need to check if the snapshot already exists.